message-me/app/views/chatrooms/index.html.erb

```diff
@@ -1 +1,62 @@
-<h1>this is index.html.erb for chatroom</h1>
+<h4 class="ui center aligned medium icon header">
+  <i class="orange circular coffee icon"></i>
+  Say Something
+</h4>
+
+<div class="ui two column grid">
+  <div class="twelve wide column">
+    <div class="ui fluid raised card chatbox">
+      <div class="content">
+        <div class="ui small feed">
+          <h4 class="ui header">Followers Activity</h4>
+          <div class="event">
+            <div class="content">
+              <div class="summary">
+                <a>Elliot Fu</a> added <a>Jenny Hess</a> as a friend
+              </div>
+            </div>
+          </div>
+          <div class="event">
+            <div class="content">
+              <div class="summary">
+                <a>Stevie Feliciano</a> added <a>Elliot Fu</a> as a friend
```

Comment:
What are these `a` tags about? No link nothing?

Reply:
Nevermind. I see that you change this in https://github.com/LeeRenJie/Learn-Ruby/pull/79